indexer/classificator.cpp

```diff
@@ -427,13 +436,11 @@ void Classificator::Clear()
 
 string Classificator::GetReadableObjectName(uint32_t type) const
 {
-  string s = classif().GetFullObjectName(type);
-
-  // remove ending dummy symbol
+  string s = GetFullObjectName(type);
+  // Remove ending dummy symbol.
   ASSERT ( !s.empty(), () );
-  s.resize(s.size()-1);
-
-  // replace separator
+  s.pop_back();
```

Comment:
Old classificator code has a lot of formatting issues. Is it ok if we reformat it’s hpp and cpp later?